modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -112,13 +123,17 @@ The possible values of MS2-Status-Code-Selection are:
 +
 `RANDOM`:: The mocking service randomly selects one of the status codes to return. For example, if the specification defines the status codes for a POST method 500, 409, 400, 200, the mocking service returns the response for any one of them.
 
+== Test API Specifications That Contain Errors
```

Comment:
Less verbose:

> Test API Specifications for Errors

Reply:
No, the section is about testing API specifications _that_ contain errors.